gapis/api/vulkan/synthetic.api

```diff
@@ -256,3 +256,16 @@ cmd void ReplayDestroyVkDebugReportCallback(
   VkDebugReportCallbackEXT callback) {
   if !(instance in Instances) { vkErrorInvalidInstance(instance) }
 }
+
+@synthetic
+cmd VkResult replayWaitForFences(
+  VkDevice device,
+  u64 count,
+  VkFence* pFences,
+  u64* pExpected,
+  bool waitAll,
+  u64 timeout) {
```

Comment:
nit: (I know this matches other functions here) Indent continuations 4 spaces, so it's easy to see where the function body starts.